modules/mobile_session_impl.lua

```diff
@@ -77,19 +110,52 @@ function mt.__index:SendRPC(func, arguments, fileName)
   return self.rpc_services:SendRPC(func, arguments, fileName)
 end
 
----Start specific service
+--- Send encrypted RPC
+-- @tparam string func RPC name
+-- @tparam table arguments Arguments for RPC function
+-- @tparam string fileName Path to file with binary data
+function mt.__index:SendEncryptedRPC(func, arguments, fileName)
+  if self.isSecuredSession and self.security:checkSecureService(constants.SERVICE_TYPE.RPC) then
+    return self.rpc_services:SendRPC(func, arguments, fileName, true)
+  end
+  error("Error: Can not send encrypted request. "
+    .. "Secure service was not established. Session: " .. self.sessionId.get())
+end
+
+--- Start specific service
 -- For service == 7 should be used StartRPC() instead of this function
 -- @tparam number service Service type
 -- @treturn Expectation expectation for StartService ACK
 function mt.__index:StartService(service)
   return self.control_services:StartService(service)
 end
 
+--- Start specific secured service
+-- @tparam number service Service type
+-- @tparam table securitySettings settings for SSL
```

Comment:
@aderiabin There is no 'securitySettings' parameter

Reply:
@dboltovskyi Done (see ab761db127418d0f025c8e5768470f626587e41c)